distributions/README.md

```diff
@@ -0,0 +1,20 @@
+# Criteria for Supported Distributions
+
+Support for a distribution implies all the duties covered by the Approver and Maintainer role requirements. In addition, support means that the Collector SIG is the owner and maintainer of the binaries/images of the different Collector distributions and is responsible for the pipeline that produces those artifacts.
+
+The Collector SIG must support at least one distribution that is recommended for production which includes support for Prometheus, Jaeger, Zipkin, and OpenCensus.
+
+Distributions supported by the Collector SIG should fulfill the following criteria:
+
+1. Serve a specific purpose and those purposes should have minimal overlap.
+2. Meet general needs and not be too niche.
+3. Only target the needs of the OpenTelemetry project.
```

Comment:
These two goals seem to be in conflict.  Do we have guidance on how to resolve the tension?  Goal 2) is also ill-defined; how am I to know how niche is "too niche"?  That's a judgement that will vary wildly based on perspective.

Reply:
The intention behind 2 is to stop us from supporting distributions that won't get a lot of use.  For example, if someone came to us and wanted us to support a distribution that targeted their specific environment needs, or if someone came and said "I think the community should support am image with only components X, Y and Z bc I need it", we could point to this rule.

The intention behind 3 is to prevent us from supporting distributions for other third-parties. The issue was raised specifically around venders. We don't intend to be the only source of collector distributions - we do intent to be a source of collector distributions that fit our needs.

When stated verbosely I don't feel they are in conflict, but I'm willing to continue wordsmithing until the intent is clearer.

How does this sound:

```
  2. Should meet general needs and be desired by many users
  3. Should not be specific to any vendor
```

@open-telemetry/collector-contrib-approvers please review.

Reply:
I like that wording more than the current one, I also [struggled to understand (3) at first](https://github.com/open-telemetry/opentelemetry-collector-releases/pull/376#discussion_r1275234338), this sounds clearer

Reply:
That sounds like a better formulation, thank you.